packages/react/src/components/Alert/Alert.tsx

```diff
@@ -179,20 +185,22 @@ class Alert extends AutoControlledComponent<WithAsProp<AlertProps>, AlertState>
             xSpacing: 'after',
           },
         })}
-        {Box.create(header, {
-          defaultProps: {
-            className: Alert.slotClassNames.header,
-            styles: styles.header,
-            ...accessibility.attributes.header,
-          },
-        })}
-        {Box.create(content, {
-          defaultProps: {
-            className: Alert.slotClassNames.content,
-            styles: styles.content,
-            ...accessibility.attributes.content,
-          },
-        })}
+        <Flex as="div" className="wrapper" {...accessibility.attributes.wrapper} id={wrapperId}>
```

Comment:
We should not use utility components in our internals, consider to create a slot for it.